plugins/modules/aci_l3out_logical_interface_profile_eigrp_profile.py

```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Jason Juenger (@jasonjuenger) <jasonjuenger@gmail.com>
+# Copyright: (c) 2023, Gaspard Micol (@gmicol) <gmicol@cisco.com>
```

Comment:
dont think Gaspard had anything to do with your code, could be removed

Reply:
removed